### ISSUE TITLE
Enable terminix to execute commands from the terminal

### ISSUE THREAD

**Issue by gnunn1** [gnunn1/terminix#40]
When using terminix, sometimes you want to perform an action in terminix and it would be faster just to be able to bang it out from the keyboard. This feature will enable users to execute any parameterless GAction from the terminix prompt.

From a design perspective, the following changes will be made:

a. Add an environment variable called TERMINIX_ID with the UUID of the terminal
b. When user executes second copy of terminix (called the remote instance in GTK Application parlance), check for a new command parameter. If found, invoke a method to execute a new Command action with the name of the command and the terminal UUID as parameters
c. Command will use same action name format string as keyboard shortcuts in GSettings, i.e. "prefix-actionname"
d. This method gets executed in the remote Application however GTK Application automatically sends the invocation of the command GAction to the primary instance
e. Remote instance terminates
f. Primary instance executes command action, find the terminal with the matching UUID and then walk up the widget chain until we find an ActionGroup that matches the prefix of the command.
f. Execute the action

In a nutshell, users can issue command as follows:

terminix -c terminal-split-horizontal
terminix -c terminal-split-vertical
terminix -c app-new-window

etc
